packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -82,14 +88,127 @@ const List = ({
   );
 };
 
+/**
+ * Generic vertically-centered Header layout
+ */
+const Header = ({
+  flexBoxProps,
+  headingProps,
+  heading,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  heading: string;
+  flexBoxProps?: ComponentProps<typeof FlexBox>;
+  headingProps?: ComponentProps<typeof Heading>;
+}) => (
+  <Slide {...rest}>
+    <FlexBox height="100%" {...flexBoxProps}>
+      <Heading {...headingProps}>{heading}</Heading>
+    </FlexBox>
+  </Slide>
+);
+
+/**
+ * Section layout with left aligned text
+ */
+const Section = ({
+  sectionTitle,
+  sectionTitleProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  sectionTitle: string;
+  sectionTitleProps?: ComponentProps<typeof Heading>;
+}) => (
+  <Header
+    heading={sectionTitle}
+    headingProps={sectionTitleProps}
+    flexBoxProps={{ justifyContent: 'flex-start' }}
+  />
+);
+
+/**
+ * Statement layout with centered text
+ */
+const Statement = ({
+  statement,
+  statementProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  statement: string;
+  statementProps?: ComponentProps<typeof Heading>;
+}) => <Header heading={statement} headingProps={statementProps} />;
+
+/**
+ * Big Fact with optional fact information
+ */
+const BigFact = ({
+  fact,
+  factInformation,
+  factProps,
+  factInformationProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  fact: string;
+  factInformation?: string;
+  factProps?: ComponentProps<typeof Text>;
+  factInformationProps?: ComponentProps<typeof Text>;
+}) => (
+  <Slide {...rest}>
+    <FlexBox>
+      <Box>
+        <Text fontSize="250px" textAlign="center" {...factProps}>
```

Comment:
250px looks good for numbers, but it's sentences overflow pretty quickly. I think this is still a reasonable default, and we can override the font size via `factProps.fontSize`, but I think it'd be nice for this to be an easily overridable prop.

Maybe we can have another (optional) prop like `factSize` that defaults to 250, but allows us to change this size just by doing `factSize={150}` or something like that?